components/org.wso2.carbon.identity.api.server.tenant.management/org.wso2.carbon.identity.api.server.tenant.management.v1/src/main/java/org/wso2/carbon/identity/api/server/tenant/management/v1/core/ServerTenantManagementService.java

```diff
@@ -193,6 +199,33 @@ public List<OwnerResponse> getOwners(String tenantUniqueID) {
         }
     }
 
+    public OwnerInfoResponse getOwner(String tenantUniqueID, String ownerID, String additionalClaims) {
+
+        try {
+            Tenant tenant = TenantManagementServiceHolder.getTenantMgtService().getTenant(tenantUniqueID);
+            if (tenant.getAdminUserId() == null || !tenant.getAdminUserId().equals(ownerID)) {
```

Comment:
Can tenant be null ?

Reply:
If tenant is not found `TenantMgtException` is thrown